File: CodeFestival/Q31-Q40/Q34_sort_구현하기.py
import copy
height = list(input().split())
tmp = copy.deepcopy(height)
height.sort()
' '.join(tmp)
' '.join(height)

if tmp == height:
    print("YES")
else:
    print("NO")    



# user_input = input()

# l = list(user_input.strip().split())
# l = [int (i) for i in l]

# if l != sorted(l):
# 	print("NO")
	
# else:
# 	print("YES")

File: CodeFestival/Q31-Q40/Q34_sort_구현하기.js
let height = prompt("키를 입력하세요.").split(' ');
let tmp = [...height];
height.sort(function(a, b) {
    return a-b;
})

tmp = tmp.join(' ');
height = height.join(' ');
console.log(tmp);
console.log(height);

if (tmp == height) {
    console.log("YES");
} else {
    console.log("NO");
}

// const unsorted = prompt('키를 입력하세요');
// let sorted = "";

// sorted = unsorted
//   .split(" ")
//   .sort(function(a, b) {
//     return a - b;
//   })
//   .join(" ");

// if (unsorted === sorted) {
//   console.log("Yes");
// } else {
//   console.log("No");
// }